### DISCUSSION TITLE
403 Forbidden

### DISCUSSION THREAD

**Discussion by maddalakum** [cypress-io/cypress#23615] [Questions and Help]
Cant able to redirect to this site . Please help me 
![image](https://user-images.githubusercontent.com/25227751/187467178-8818b3af-5a62-4ed6-ba82-7e4d1075170a.png)

**Comment by palamarchukser** [comment]
Hey @maddalakum, hope this helps 
```
cy.visit('https://signin.infusionsoft.com/', {
    failOnStatusCode: false,
})
```
